runtime/lang/menu_ru_ru.utf-8.vim

```diff
@@ -1,337 +1,1266 @@
-    tmenu ToolBar.Shell					Оболочка
-    tmenu ToolBar.RunCtags				Создать файл меток
-    tmenu ToolBar.TagJump				Перейти к метке
-    tmenu ToolBar.Help					Справка
-    tmenu ToolBar.FindHelp				Найти справку
+    let did_toolbar_tmenu = 1
+    tmenu ToolBar.Open Открыть файл
+    tmenu ToolBar.Save Сохранить файл
+    tmenu ToolBar.SaveAll Сохранить все файлы
+    tmenu ToolBar.Print Напечатать
+    tmenu ToolBar.Undo Отменить
+    tmenu ToolBar.Redo Вернуть
+    tmenu ToolBar.Cut Вырезать
+    tmenu ToolBar.Copy Копировать
+    tmenu ToolBar.Paste Вклеить
+    if !has("gui_athena")
+      tmenu ToolBar.Replace Найти или заменить...
+      tmenu ToolBar.FindNext Найти следующее
+      tmenu ToolBar.FindPrev Найти предыдущее
+    endif
+    tmenu ToolBar.LoadSesn Загрузить сеанс редактирования
+    tmenu ToolBar.SaveSesn Сохранить сеанс редактирования
+    tmenu ToolBar.RunScript Выполнить сценарий Vim
+    tmenu ToolBar.Make Компиляция
+    tmenu ToolBar.RunCtags Создать файл меток
+    tmenu ToolBar.TagJump Перейти к метке
+    tmenu ToolBar.Help Справка
+    tmenu ToolBar.FindHelp Найти справку
   endfun
 endif
-"
-"
-" Dialog texts
-"
-" Find in help dialog
-"
-let g:menutrans_help_dialog = "Введите команду или слово для поиска:\n\nДобавьте i_ для поиска команд режима Вставки (например, i_CTRL-X)\nДобавьте c_ для поиска команд Обычного режима (например, с_<Del>)\nДобавьте ' для поиска справки по опции (например, 'shiftwidth')"
-"
-" Searh path dialog
-"
-let g:menutrans_path_dialog = "Укажите путь для поиска файлов.\nИмена каталогов разделяются запятыми."
-"
-" Tag files dialog
-"
-let g:menutrans_tags_dialog = "Введите имена файлов меток (через запятую).\n"
-"
-" Text width dialog
-"
-let g:menutrans_textwidth_dialog = "Введите ширину текста для форматирования.\nДля отмены форматирования введите 0."
-"
-" File format dialog
-"
-let g:menutrans_fileformat_dialog = "Выберите формат файла."
-let g:menutrans_fileformat_choices = "&Unix\n&Dos\n&Mac\nО&тмена"
-"
-let menutrans_no_file = "[Нет файла]"
+" }}}
+
+" Syntax menu
+menutrans &Syntax &Синтаксис
+" Syntax menuitems {{{1
+menutrans &Show\ File\ Types\ in\ Menu Показать\ меню\ выбора\ типа\ &файла
+menutrans &Off &Отключить
+menutrans &Manual Вру&чную
+menutrans A&utomatic &Автоматически
+menutrans On/Off\ for\ &This\ File Вкл/выкл\ для\ &этого\ файла
+menutrans Co&lor\ Test Проверка\ &цветов
+menutrans &Highlight\ Test Проверка\ под&светки
+menutrans &Convert\ to\ HTML С&делать\ HTML\ с\ подсветкой
+
+" From synmenu.vim
+menutrans Set\ '&syntax'\ Only &Изменять\ только\ значение\ 'syntax'
+menutrans Set\ '&filetype'\ Too Изменять\ &также\ значение\ 'filetype'
+" menutrans AB TRANSLATION\ MISSING
+" menutrans A2ps\ config TRANSLATION\ MISSING
+" menutrans Aap TRANSLATION\ MISSING
+" menutrans ABAP/4 TRANSLATION\ MISSING
+" menutrans Abaqus TRANSLATION\ MISSING
+" menutrans ABC\ music\ notation TRANSLATION\ MISSING
+" menutrans ABEL TRANSLATION\ MISSING
```

Comment:
I don't think filetypes should be translated. Even though  few words could, in principle, be. But it looks more natural to me just leave them all as is.

Reply:
> I don't think filetypes should be translated. Even though few words could, in principle, be. But it looks more natural to me just leave them all as is.

Thanks. Only few of them could to be translated. It's not uniform in different language. So I've listed them all, and let people of each language decide for themselves. So just keep the items untranslated if you think it should be translated. I'll clean it by regular expressions finally.

Reply:
> I don't think filetypes should be translated. Even though few words could, in principle, be. But it looks more natural to me just leave them all as is.

Thanks. Only few of them could be translated. It's not uniform in different language. So I've listed them all, and let people of each language decide for themselves. So just keep the items untranslated if you think it shouldn't be translated. I'll clean them by regular expressions finally.